resources/views/layouts/_footer.blade.php

```diff
@@ -7,7 +7,7 @@
             </div>
         </div>
     </div>
-
-    <privacy-info privacy-policy-url="{{ url(option('general', 'cookies_policy_url', '')) }}"></privacy-info>
+    @php($policyUrl = option('general', 'cookies_policy_url'))
+    <privacy-info privacy-policy-url="{{ ($policyUrl) ? urlMl($policyUrl, app()->getLocale()) : null}}"></privacy-info>
```

Comment:
The following is trivially always true: 
```urlMl($url, $currentLocale) == url($url)```

Use just the `url` helper to make the code simpler to read